src/core/Typewriter.js

```diff
@@ -102,6 +107,18 @@ class Typewriter {
     this.state.elements.cursor.innerHTML = this.options.cursor;
     this.state.elements.container.innerHTML = '';
 
+    // initial text
+    let txt = this.options.initialText;
+    if(txt)
+      txt.split('').forEach(s => {
+        const textNode = document.createTextNode(s);
+        this.state.elements.wrapper.appendChild(textNode);
+        this.state.visibleNodes.push({
+          type: VISIBLE_NODE_TYPES.TEXT_NODE,
+          node: textNode,
+        });
+      });
```

Comment:
Split `options.initialText` into single characters, append as individual text nodes, add to `state.visibleNodes`.